### PR TITLE
fixed issue #397 get rid of horizontal scrolling on mobile.

### PR DESCRIPTION
**Pull request by Chaitanyassr**
before
after

### CODE REVIEW

**Comment by mariobehling** [comment]
Please provide a link to a test deployment.

File: index.html [hunk 1]
```diff
@@ -644,9 +645,8 @@ <h1>How Can I Participate in FOSSASIA projects?</h1>
 </html>
 
 <style>
-#results_projects {
```
Comment (anishagg17) [inline index.html]: I don't think this has anything related to issue

Comment (Chaitanyassr) [reply index.html]: yes it is not related to the issue. i guess have re written the 647th line thats why it is in red.i m just removing those extra spaces.

Comment (Chaitanyassr) [reply index.html]: yup, this line was already present.this arrised because i left an empty line after 647. i have removed it in latest commit.

**Comment by Chaitanyassr** [comment]
> Please provide a link to a test deployment.
link--
https://chaitanyassr.github.io/labs.fossasia.org

**Comment by Chaitanyassr** [comment]
> Issue still persists on test site in a number of resolutions. Please compare with FOSSASIA.org site as well.

okay.

**Comment by Chaitanyassr** [comment]
> Issue still persists on test site in a number of resolutions. Please compare with FOSSASIA.org site as well.

Sir , here's the new deployed link-
https://chaitanyassr.github.io/labs.fossasia.org

**Comment by mariobehling** [comment]
Thanks. Works for me here.

**Pull request merged**